test/dotnet-run.Tests/RunTests.cs

```diff
@@ -20,7 +20,7 @@ public void RunsSingleTarget()
             TestInstance instance = TestAssetsManager.CreateTestInstance(Path.Combine(RunTestsBase, "TestAppFullClr"))
```

Comment:
Please, add a test where we actually see the additional probing path being added for non desktop apps.